cloud-controller-manager/cmd/digitalocean-webhook-server/main.go

```diff
@@ -47,15 +47,15 @@ func startWebhookServer() error {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "failed to retrieve kubeconfig %v\n", err)
 	}
-	server := webhook.Server{}
+	server := webhook.NewServer(webhook.Options{})
 	c, err := client.New(config, client.Options{Scheme: scheme})
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "failed to construct cr client %v\n", err)
 	}
 
 	server.Register("/validate-doks-lb-service", &webhook.Admission{Handler: &do.DOKSLBServiceValidator{Client: c, Log: ll}})
 	ll.Info("Registering Webhook server handlers")
-	if err = server.StartStandalone(ctrl.SetupSignalHandler(), scheme); err != nil {
+	if err = server.Start(signals.SetupSignalHandler()); err != nil {
```

Comment:
Based of  https://github.com/kubernetes-sigs/controller-runtime/pull/2134/files#diff-08e2705e87e1569d1eb9d7e815400fc74bfaf9ed902d4a4e38bddbb13b3e43a2R99